metadataservice/test/testing.py

```diff
@@ -1,13 +1,11 @@
 import uuid
-from metadataservice.client.api import server_connect#, db_disconnect
-from metadataservice.client import conf
 from copy import deepcopy
 
 conn = None
 testing_config = {
     'database': "mds_testing_disposable_{}".format(str(uuid.uuid4())),
     'host': 'localhost',
-    'port': 27017,
+    'port': 7770,
```

Comment:
We had better document this. The team will need to know this new port number by heart.